library/stdtypes.po

```diff
@@ -446,14 +451,12 @@ msgstr ""
 "spécifique."
 
 #: ../Doc/library/stdtypes.rst:268
-#, fuzzy
 msgid ""
 "All numeric types (except complex) support the following operations (for "
 "priorities of the operations, see :ref:`operator-summary`):"
 msgstr ""
-"Tous les types numériques (sauf complexe) gèrent les opérations suivantes, "
-"classées par priorité ascendante (toutes les opérations numériques ont une "
-"priorité plus élevée que les opérations de comparaison) :"
+"Tous les types numériques (sauf complexe) gèrent les opérations suivantes "
+"(pour les priorités sur les opérations, voir :ref:`operator-summary`) :"
```

Comment:
```suggestion
"(pour les priorités sur les opérations, voir :ref:`operator-summary`) :"
```

/tmp/padpo_j77_iede/library/stdtypes.po: 453 [     NBSP     ] There should be a non-breakable space before ":": between ###ns, voir « operator-summary ») ### and ###:###